code/modules/mob/living/carbon/carbon.dm

```diff
@@ -271,6 +271,9 @@
 /mob/living/carbon/is_muzzled()
```

Comment:
Does both this check & the new check below fully function? Hopefully the dual definitions play nice with eachother.